tool-testsuite/test/org/antlr/v4/test/tool/TestSymbolIssues.java

```diff
@@ -315,51 +330,12 @@ public void testSetUp() throws Exception {
 				"B: 'b';\n" +
 				"C: 'c';\n",
 
-				"error(" + ErrorType.LABEL_TYPE_CONFLICT.code + "): L.g4:8:15: label t2=b type mismatch with previous definition: t2=a\n" +
-				"error(" + ErrorType.LABEL_TYPE_CONFLICT.code + "): L.g4:11:17: label t3+=c type mismatch with previous definition: t3+=a\n"
-		};
-
-		testErrors(test, false);
-	}
-
-	// https://github.com/antlr/antlr4/issues/1543
```

Comment:
:bulb: I would prefer to keep this and the following as distinct tests since they target a specific issue.